packages/suite-desktop/src/support/screens/TorLoadingScreen.tsx

```diff
@@ -0,0 +1,262 @@
+import React, { useState, useEffect } from 'react';
+import styled, { css } from 'styled-components';
+
+import { desktopApi, BootstrapTorEvent } from '@trezor/suite-desktop-api';
+import { LoadingScreen } from '@suite-support/screens/LoadingScreen';
+import { P, Button, Progress, variables } from '@trezor/components';
+import { Image } from '@suite-components';
+import { ThemeProvider } from '@suite-support/ThemeProvider';
+import { TorStatus } from '@suite-types';
+
+const Wrapper = styled.div`
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 16px;
+`;
+
+const Text = styled(P)`
+    height: 0;
+    font-size: ${variables.FONT_SIZE.H2};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    color: ${props => props.theme.TYPE_DARK_GREY};
+`;
+
+const ModalWindow = styled.div`
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    border-radius: 16px;
+    text-align: center;
+    transition: all 0.3s;
+    max-width: 95%;
+    min-width: 305px;
+    width: 720px;
+    padding: 42px;
+
+    ${({ theme }) =>
+        css`
+            background: ${theme.BG_WHITE};
+            box-shadow: 0 10px 80px 0 ${theme.BOX_SHADOW_MODAL};
+        `}
+`;
+
+const Percentage = styled.div`
+    display: flex;
+    font-variant-numeric: tabular-nums;
+    height: 24px;
+`;
+
+const StyledProgress = styled(Progress)`
+    display: flex;
+    margin-right: 20px;
+    margin-left: 20px;
+    border-radius: 5px;
+    background: ${({ theme }) => theme.STROKE_GREY_ALT};
+    flex: 1;
+
+    ${Progress.Value} {
+        height: 3px;
+        position: relative;
+        border-radius: 5px;
+    }
+`;
+
+const ProgressWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    background: #e8e8e8;
```

Comment:
Please use value from theme and check it in dark color scheme
<img width="725" alt="image" src="https://user-images.githubusercontent.com/3729633/182572157-a46af165-5f82-4f45-96f4-6e07f77efa8b.png">

Reply:
Thank  you for that detail! it should be fixed now.